bin/exec

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd "$(dirname "$0")/.."
+
+if [ -z "$2" ]; then
+    docker exec -ti "daffy-${1}" /bin/bash
```

Comment:
When using docker-compose we can execute command with `docker-compose exec ${1} /bin/bash` without forcing the container_name.

Reply:
But container name is useful when you want to use one container in the other. I've never used docker-compose exec but it seems to be doing exactly the same thing as my script